service/src/main/java/bio/terra/landingzone/service/iam/LandingZoneSamService.java

```diff
@@ -134,9 +134,13 @@ public void createLandingZone(BearerToken bearerToken, UUID billingProfileId, UU
     var policies =
         Map.of(
             "owner",
-            new AccessPolicyMembershipV2()
-                .addMemberEmailsItem(userInfo.getUserEmail())
-                .addRolesItem(SamConstants.SamRole.OWNER));
+                new AccessPolicyMembershipV2()
+                    .addMemberEmailsItem(userInfo.getUserEmail())
+                    .addRolesItem(SamConstants.SamRole.OWNER),
+            "user",
+                new AccessPolicyMembershipV2()
+                    .memberEmails(samConfig.getLandingZoneResourceUsers())
```

Comment:
Shouldn't this be added only if the configuration has value(s). Or if we expect this to be required throw an exception if not they are not set?

Reply:
I think we can assume Leo SA will always need to exist there

Reply:
Mm, it might good to be defensive in case the list is empty or null. I can update

Reply:
Done in [latest commit](https://github.com/DataBiosphere/terra-landing-zone-service/pull/39/commits/73c920cfe66f7fde75829f7446733c3651182b9d)